h/static/scripts/annotator/plugin/pdf-metadata.js

```diff
@@ -71,4 +73,13 @@ function fingerprintToURN(fingerprint) {
   return 'urn:x-pdf:' + String(fingerprint);
 }
 
+function getPDFURL(app) {
+    // Local file:// URLs should not be saved in document metadata.
+    // Entries in document.link should be URIs. In the case of
+    // local files, omit the URL.
+    if (app.url.indexOf('file://') !== 0) {
```

Comment:
I prefer that if functions return a value through one path, then they explicitly return a value through all paths rather than just relying on the implicit `return undefined`. That makes it obvious that any null values returned are intentional.

In short, just add `return null` at the end here.